### PR TITLE
Resolve #192 search not case insensitive

### LINKED ISSUE #192: How to make Solr case insensitive
Is there a way to make Solr searches case insensitive?
It seems to be dependant on wether the filesystem is case sensitive or not.
Our local test environment is case insensitive, yet production server is, thus causing problems.

Right now, we've come up with a quick fix for this issue by adding extra altered searchterms. 
I am wondering if there is a better way to do this?
    
```
$query = new BaseQuery();
$query->setStart($start);
$query->setRows(12);
$query->setSpellcheck(true);

// TMP CASE INSENSITIVE FIX
$query->addTerm(strtoupper($searchTerm));
$query->addTerm(ucfirst($searchTerm));
$query->addTerm(strtolower($searchTerm));
```

Maybe someone can explain how to do this properly.

### PR DESCRIPTION
**Pull request by Firesphere**
No immediate need for a clean re-index, as the query parts applied already do the job, it seems.

### CODE REVIEW

**Comment by Firesphere** [comment]
Seems to not fully solve the issue (yet?)

**Comment by Firesphere** [comment]
Update: My changes have not properly been put in to the second test index, thus the second index is indeed still wrong on my test environment. Proceed as it was, this should fix the issue.

File: Solr/5/templates/types.ss [hunk 1]
```diff
@@ -367,7 +367,7 @@
 <fieldType name="lowercase" class="solr.TextField" positionIncrementGap="100">
     <analyzer>
         <tokenizer class="solr.KeywordTokenizerFactory"/>
-        <filter class="solr.LowerCaseFilterFactory"/>
+        <filter class="solr.`LowerCase`FilterFactory"/>
```
Comment (marczhermo) [inline Solr/5/templates/types.ss]: Looks like stray ` ` `
```suggestion
        <filter class="solr.LowerCaseFilterFactory"/>
```

Comment (Firesphere) [reply Solr/5/templates/types.ss]: Yaps, that is absolutely an accidental backtick on my end!

**Pull request merged**